lib/utils/alwaysDisplayError.ts

```diff
@@ -1,5 +1,5 @@
-module.exports = function(err) {
+export = function(err: any) {
```

Comment:
```suggestion
export = function(err: NightwatchError) {
```

Reply:
Same as the first comment, since we are checking in the next line whether the `err` argument is an instance of `Error` or not, that means that `err` could be anything. But we should probably use `unknown` instead of `any`.

Reply:
Changed it to `unknown`.